nntrainer/models/neuralnet.h

```diff
@@ -443,6 +443,11 @@ class NeuralNetwork : public ml::train::Model {
    */
   void disableDynamicFineTuning() { dynamic_training_opt.disable(); }
 
+  /**
+   * @brief     Update batch size of the model as well as its layers/dataset
+   */
+  void setBatchSize(unsigned int batch_size);
```

Comment:
please use setProperty than exposing this to public.

Reply:
Done